client/packages/host/src/routers/ManageRouter.tsx

```diff
@@ -6,33 +6,29 @@ const NameService = React.lazy(
   () => import('@openmsupply-client/system/src/Name/Service')
 );
 
-const IndicatorsDemographicsService = React.lazy(
-  () =>
-    import(
-      '@openmsupply-client/system/src/IndicatorsDemographics/Service/Service'
-    )
+const ManageService = React.lazy(
+  () => import('@openmsupply-client/system/src/Manage/Service')
```

Comment:
Can you please comment about this change, why it was done, etc.. (since PR is quite big already), happy to review this but a quick note 'This is renaming, this functionality is now 'common' etc..

Reply:
Sorry did put comments as above but looks like they became outdated somehow, will add again 🙏